rllib/contrib/random_agent/random_parametric_agent.py

```diff
@@ -0,0 +1,84 @@
+from abc import ABC
+
+import ray
+
+import numpy as np
+
+from ray.rllib import Policy
+from ray.rllib.agents import with_common_config
+from ray.rllib.agents.trainer_template import build_trainer
+from ray.rllib.evaluation.worker_set import WorkerSet
+from ray.rllib.execution.metric_ops import StandardMetricsReporting
+from ray.rllib.execution.rollout_ops import ParallelRollouts, SelectExperiences
+from ray.rllib.examples.env.parametric_actions_cartpole import ParametricActionsCartPole
+from ray.rllib.models.modelv2 import restore_original_dimensions
+from ray.rllib.utils import override
+from ray.rllib.utils.typing import TrainerConfigDict
+from ray.util.iter import LocalIterator
+from ray.tune.registry import register_env
+
+DEFAULT_CONFIG = with_common_config({})
+
+
+class RandomParametriclPolicy(Policy, ABC):
+    """
+    Just pick a random legal action
+    The outputted state of the environment needs to be a dictionary with an 'action_mask' key
+    containing the legal actions for the agent
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.exploration = self._create_exploration()
+
+    @override(Policy)
+    def compute_actions(self,
+                        obs_batch,
+                        state_batches=None,
+                        prev_action_batch=None,
+                        prev_reward_batch=None,
+                        info_batch=None,
+                        episodes=None,
+                        **kwargs):
+
+        obs_batch = restore_original_dimensions(np.array(obs_batch, dtype=np.float32), self.observation_space, tensorlib=np)
```

Comment:
Looks like `format.sh` has not been run yet.

Reply:
I've tried to run it, but has I've said in the comment, I got his error:
ERROR: ./scripts/format.sh: line 44: YAPF_DOWNLOAD_COMMAND_MSG: unbound variable

Reply:
Problem solved
./script/format.sh run without error
But the hooks say that **ci/travis/format.sh** encountered an error because it doesn't find yapf, even if it is already installed with the right version
So, I think it's a problem with my machine and hope Travis will not encounter the same issue.